lib/src/code_field/code_controller.dart

```diff
@@ -498,7 +498,12 @@ class CodeController extends TextEditingController {
 
     // Return parsing
     if (_language != null) {
-      return _processLanguage(text, CodeTheme.of(context), style);
+      return const SpanBuilder().build(
+        code: _lastCode,
+        theme: _getTheme(context),
+        textStyle: style,
+      );
+      //return _processLanguage(text, CodeTheme.of(context), style);
```

Comment:
Is this commented code needed?

Reply:
We may need to compare output to the old renderer if any highlighting bugs show up.